resources/testharness.js

```diff
@@ -4296,81 +4287,62 @@
         });
 
         function get_asserts_output(test) {
+            const asserts_output = render(
+                ["details", {},
+                    ["summary", {}, ""],
+                    ["table", {}, ""] ]);
+
             var asserts = asserts_run_by_test.get(test);
             if (!asserts) {
-                return "No asserts ran";
+                asserts_output.querySelector("summary").textContent = "No asserts ran";
+                return asserts_output;
             }
-            rv = "<table>";
-            rv += asserts.map(assert => {
-                var output_fn = "<strong>" + escape_html(assert.assert_name) + "</strong>(";
-                var prefix_len = output_fn.length;
-                var output_args = assert.args;
-                var output_len = output_args.reduce((prev, current) => prev+current, prefix_len);
-                if (output_len[output_len.length - 1] > 50) {
-                    output_args = output_args.map((x, i) =>
-                    (i > 0 ? "  ".repeat(prefix_len) : "" )+ x + (i < output_args.length - 1 ? ",\n" : ""));
-                } else {
-                    output_args = output_args.map((x, i) => x + (i < output_args.length - 1 ? ", " : ""));
-                }
```

Comment:
I dropped that on purpose, since this appears to be dead code:

* assert_args is an array with the arguments passed to the original `AssertRecord`, formatted by `format_value`.
* The last argument seems to always be a string.
* Since `format_value` with put strings in quotes, the last character of the last string is always `"`.
* Reducing this over `+` will produce a string whose last character is `"`.
* Comparing that last character against 50 will always be `false`.

Example:
- `assert_equals(2, 3, "explainy explainy")`
- => `[2,3, '"explainy explainy"' ]`
- => '23"explainy explainy"'

I *think* that is meant to do line breaking. But I don't think it actually does this.

I tried to rewrite this to do line breaking. But firstly, that'd be a change from what we had, and secondly, that looked worse than just letting the browser do the line breaking. 

Maybe I'm misunderstanding this; but in that case I will need a bit of a hint of where I went wrong.

Original discussion here: https://chromium-review.googlesource.com/c/chromium/src/+/5290877/comment/b3ec4e27_c7bdddba/

Reply:
I don't think "The last argument seems to always be a string." is correct, but indeed `output_len` is a string and I'm pretty sure there's no single character that can be `> 50`. I imagine
```js
var output_len = output_args.reduce((prev, current) => prev+current.length, prefix_len);
if (output_len > 50) {
```
was meant, to achieve an output like
```js
assert_equals("looooooooooooooooooooooooooooooong string",
              "looooooooooooooooooooooooooooooong string")
```
However, it doesn't indent the following lines by the length of `assert_equals(` but the length of `<strong>assert_equals</strong>(`, which means that the rendered code doesn't line up anyway.

Since nobody noticed or cared enough to figure out what was wrong with this code in the three years since it was added in #27224, I think you're justified in removing it.